### DISCUSSION TITLE
0 value don'w show in tool tips

### DISCUSSION THREAD

**Discussion by n-arakawa** [marcusolsson/grafana-gantt-panel#85] [Ideas]
In tool tips, labelFields with 0 (or any falsy values) does not show because of filter.

- Hope to show meaningful data such as  0, ""(empty string), false and NaN
- null and undefined : convert to "" to keep order of the labelFields

How about adding **?? ""** ?

src/GanttChart.tsx line 332
```
                    .filter((field) => field?.values.get(i) ?? "" )

```

**Comment by marcusolsson** [comment]
Thanks for the feedback! What do you mean by "the order of the labelFields"? I agree that we should fix 0, false etc, but I don't see the purpose of displaying null and undefined? 

Replacing undefined and null with empty string will look like this:

<img width="252" alt="Screenshot 2021-10-13 at 15 03 43@2x" src="https://user-images.githubusercontent.com/8396880/137138007-0f6ffede-66d0-4a4b-961c-2444173e0686.png">

... which I'm not sure is useful.

Would this work for you instead?

```
.filter((field) => field?.values.get(i) !== undefined && field?.values.get(i) !== null)
```

**Comment by n-arakawa** [comment]
"the order of the labelFields"

For example there are two fields(say ,'X' and 'Y') as Numbers,

I can't distinguish (X=Null, Y=1) and (X=1, Y=Null) because tips shows only "1".
